examples/android-emulator.ts

```diff
@@ -16,7 +16,7 @@ import { parseAppMeta, pause, platformApi } from '../src/index';
     const appPath = process.argv[3] || '/path/to/app-files';
     const snapshotName = process.argv[4] || 'your-snapshot';
     const caCertPath = process.argv[5];
-    const wireguardConfigPath = process.argv[5];
+    const wireguardConfigPath = process.argv[6];
```

Comment:
While you’re at it, you should also change the comment above to document the example correctly.